app/assets/javascripts/components/tickets/reply.jsx

```diff
@@ -11,9 +11,20 @@ export const Reply = ({ message }) => {
   const failedTime = moment(message.details.delivery_failed).format('YYYY/MM/DD h:mm a');
   const failed = `Failed on ${failedTime}`;
 
+  let subject;
+  if (message.details.subject) {
+    subject = (
+      <React.Fragment>
+        <h4 className=".message-subject">{ message.details.subject }</h4>
```

Comment:
Did you mean for this to be `.message-subject` as opposed to `message-subject`?

Reply:
No. Don't need the class at all, actually. Whoops.